variants/sparkFun_pro_micro_esp32c3/pins_arduino.h

```diff
@@ -0,0 +1,58 @@
+#ifndef Pins_Arduino_h
```

Comment:
Fix variant folder name to lowercase.

Reply:
Fixed!

Reply:
Compilation still fails. The folder name was not changed in the last commit. PTAL @edspark

Reply:
Actually fixed this time, I'm not sure why it didn't take the first time.